jupyter/trustyai/ubi8-python-3.8/Dockerfile

```
@@ -0,0 +1,33 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+LABEL name="odh-notebook-jupyter-trustyai-ubi8-python-3.8" \
+    summary="Jupyter trustyai notebook image for ODH notebooks" \
+    description="Jupyter trustyai notebook image with base Python 3.8 builder image based on UBI9 for ODH notebooks" \
```

Typo: certain lines mention UBI 9 instead of UBI8
```suggestion
    description="Jupyter trustyai notebook image with base Python 3.8 builder image based on UBI8 for ODH notebooks" \
```

---

thanks for this review
will address this in separate PR.